docs/containers.conf.5.md

```diff
@@ -405,6 +405,10 @@ Example:  "/db:/var/lib/db:ro".
 The `network` table contains settings pertaining to the management of CNI
 plugins.
 
+Note: If Podman is running in a virtual machine using `podman machine` (this
+includes Mac and Windows hosts), ensure that you add the configuration in the
+virtual machine by using `podman machine ssh`.
```

Comment:
Note this does not only effect network settings, almost all option from this file must be set server side (in the machine). Notable exceptions are [machine], [farms], [engine.service_destinations], compose_providers, compose_warning_logs, and the remote option and likely more we don't even know about.

So I think this should likely be reworded and put as part of the main description. (Overall we as a team are currently trying to figure out how this whole situation can be improved)

Reply:
I'm not sure how to rephrase this to ensure that the information isn't overlooked. My thought was that including such a note directly within the configuration type would make sense.

Reply:
How should we move forward with this change?

Reply:
Is there any update on this topic? What is preventing us from moving forward with merging this change?

Reply:
I don't have a good answer, IMO putting it just under network might also be confusing for others because it affects most options.

Reply:
I attempted to relocate the information to a different location within the documentation. What are your thoughts on this approach?